### DISCUSSION TITLE
transparent corner backgrounds of widgets

### DISCUSSION THREAD

**Discussion by Ukdell** [TomSchimansky/CustomTkinter#1052] [Q&A]
I am working in a project and making a dekstop application and i want my button's corner to be transparent
![asdasd](https://user-images.githubusercontent.com/122734853/212546615-65805e9a-223c-4164-b737-dc3e9476c470.png)

I am adding a background in those widgets so i can't just define `bg_color` as solid colors like "green" or "red".
I just want them to be transparent.
any suggestions or or I request developers to fix that.
I know we have "transparet" in `fg_color`, but it should also be in `bg_color`

**Comment by TomSchimansky** [comment]
I think you are doing something wrong, the background color (bg_color) should be transparent by default. You don't have to do anything with the bg_color attribute. Are you using the latest version?

**Reply by Ukdell** [reply]
sadly `bg_color` is't transparent, it shows white color in the background
aand yes i am using 5.0.3
